LiskHQ/lisk-sdk
# Introduce advertiseAddress flag for peers - Closes #3661

### What was the problem?
Every peer is advertised to rest of the network. If a peer intended to not `advertised` across the network it wasn't possible.
### How did I solve it?
Introducing `advertiseAddress` flag, which is set to `true` by default. if a delegate/node operator wants to not advertise they can update the flag to `false`
### How to manually test it?
- Start three nodes (node1, node2, node3)
- Update `advertiseAddress` flag to `false` on node1, use node2 as the seed node
- Verify if the node3 doesn't send the node1 peer info
### Review checklist

- [ ] The PR resolves #3661
